homeassistant/components/actron_air/strings.json

```
@@ -25,5 +25,21 @@
         "title": "Actron Air OAuth2 Authorization"
       }
     }
+  },
+  "entity": {
+    "switch": {
+      "away_mode": {
+        "name": "Away Mode"
+      },
+      "continuous_fan": {
+        "name": "Continuous Fan"
+      },
+      "quiet_mode": {
+        "name": "Quiet Mode"
+      },
+      "turbo_mode": {
+        "name": "Turbo Mode"
```

These should all be sentence-cased to match Home Assistant guidelines:

```suggestion
      "away_mode": {
        "name": "Away mode"
      },
      "continuous_fan": {
        "name": "Continuous fan"
      },
      "quiet_mode": {
        "name": "Quiet mode"
      },
      "turbo_mode": {
        "name": "Turbo mode"
```